applications/machine_learning/remote/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ endif()
 ################################################################################
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-set(REMOTE_CONFIG_DIR "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${NORMALIZED_BOARD_TARGET}")
-string(CONFIGURE ${APPLICATION_CONFIG_DIR} APPLICATION_CONFIG_DIR)
+set(APP_CONFIG_DIR "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${NORMALIZED_BOARD_TARGET}")
```

Comment:
we already have something named `APPLICATION_CONFIG_DIR`, so introducing an `APP_CONFIG_DIR` has a risk of causing confusion.

Reply:
Ranamed to REMOTE_INC_DIR